### ISSUE TITLE
Slideshow content container behavior enhancement

### ISSUE THREAD

**Issue by melissaperreault** [Shopify/dawn#1339]
**Describe the current behavior**

It is not easy for merchants to build a content free slideshow. We are currently [relying on multiple settings](https://screenshot.click/22-02-bdumn-n73mo.mp4) to completely hide the  content container.

1. Remove each text input content and button labels from each slide
2. Uncheck `[ ] Show container on desktop` setting from each slide
3. Uncheck `[ ] Show content below images` on mobile on the section
4. That's a lot!

It is also not intuitive [to find out why we are left with an extra space](https://screenshot.click/22-02-g1dtp-sn7qz.png) on mobile.

**Describe the expected behavior**

We should be able to hide the content container by relying with just number 1 above. There is no benefit at showing an empty box.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 3.0
- Chrome Version 97.0.4692.99
- macOS Version 12.2

Pings here:  @wiktoriaswiecicka @danielvan

**Comment by danielvan** [comment]
Does that mean that doing 1 would by default have the same result as doing 1, 2 and 3?

**Comment by danielvan** [comment]
I do like the idea of allowing media-free slides though by using our theme colours, as I can see how a merchant could creatively use 1st slide to promote a product, 2nd slide to talk about a review and 3rd slide for collection – with the added value of the multiple layout options from slideshow. @melissaperreault happy to chat on this as I see your point on how it could overlap with multicolumn. Maybe there are use cases for both, or we can prioritise one over the other.

**Comment by bredowmax** [comment]
> > Does that mean that doing 1 would by default have the same result as doing 1, 2 and 3?
> 
> @danielvan Exactly!
> 
> > I'd love to be able to build not only content-free slideshows - but also image-free slideshows (without workarounds)
> 
> @bredowmax This is something you can already achieve with Multicolumn section, but only on mobile when you activate `Enable swipe on mobile`. Thanks for the feedback, I'll share with the team! In the meantime, I encourage you to open an issue to express your thoughts further. 🙏

Correct, the Multicolumn section does provide a similar look&feel - but I do prefer the Slideshow section in this case because of the autorotate feature. I noticed in user testing that in a Multicolumn section with "Swipe on Mobile", any but the first column is often overlooked. That's why I'd like to be able to build Slideshow slides without images.
